pype/plugins/maya/create/create_review.py

```diff
@@ -21,4 +21,6 @@ def __init__(self, *args, **kwargs):
         for key, value in animation_data.items():
             data[key] = value
 
+        data["keepImages"] = False
```

Comment:
Can we please make it class attribute? It will be the default will be configurable then. 

Thanks

Reply:
Done